stockflux-securities-master/src/components/Alert.css

```diff
@@ -0,0 +1,11 @@
+.securities-message {
+  font-size: var(--font-xxxlarge);
+}
+
+.error {
+  color: red;
+}
+
+.success {
+  color: greenyellow;
```

Comment:
Did you mean to change this colour? It should probably refer to a variable. You probably don't want the browser default `greenyellow` colour.

Reply:
Okay I've added the css variables error-red and success-green to the root of the securities master idex.css. 

I tried adding them to the main file where the rest are defined but it doesn't seem to respond. Do you know why? I think it's got something to do with the build.

Reply:
Okay I've rebuild the components folder and it now works. I've given the variables hexcodes for colors very similar to how they were before.